Model/DutyCalculator.php

```diff
@@ -278,6 +279,7 @@ public function fillOutQuoteAndSessionUsingFeeReturned($duty, $address, $apply,
             $quote->setDhlBreakdown(json_encode($response['feeTotals']));
             $this->checkoutSession->setDhlBreakdown($quote->getDhlBreakdown());
             $this->checkoutSession->setApply(true); //checkbox selection /corresponding passed value
+            $this->checkoutSession->setDhlQuoteId($response['quoteId']);
```

Comment:
The comment on line 260 says: `//DHL quoteID is already saved into session; so not doing it here again` , if we want to save the quote into the session again (which I don't mind, it should be the same value and if not then this is a good chance to correct that) could you delete this comment if it makes sense to do so?

Reply:
Hello Lizzy, Shall have a look/cleanup once my system is not `in transition` state. Right now updating few things on my dev setup. Thanks!

Reply:
sounds good! take your time

Reply:
```
//DHL quoteID is already saved into session; so not doing it here again
        $quote = $this->checkoutSession->getQuote();
```
I can remove the comment as it is not conveying what I wanted to convey and creating more confusions.
What I wanted to convey are the following:
A developer may ask down the road as to why we are saving almost everything except `quote`  in session as far as this method is concerned/involved.

As $this->checkoutSession already has $quote saved in it when we are entering this method we are not doing $this->checkoutSession->setQuote(<instance_of_quote>) to save quote the way we are saving other variables by calling $this->checkoutSession->setBlahblah(...)

If we want to save `quote` in session within this method then we have to add one extra argument to the function signature which is kind of redundant/extraneous.
Thanks!

Reply:
commit 15153627c4dedfebbb8417849bfb680a56b6e1f5 
Thank you. 
On  merging this back to magento-25 branch (as that is cut from this magento-117 branch)

Reply:
yay! thanks!